src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -625,6 +625,37 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
       .jsonPath("$.paragraphDescription").isEqualTo("")
   }
 
+  @Test
```

Comment:
The trouble is there are about 20 version of the draftAdj json response that should have this new value checked.

Look for "incidentRole"